app/cli/cmd/attestation_add.go

```diff
@@ -138,3 +147,60 @@ func newAttestationAddCmd() *cobra.Command {
 
 	return cmd
 }
+
+// displayMaterialInfo prints the material information in a table format.
+func displayMaterialInfo(status *action.AttestationStatusMaterial, policyEvaluations []*action.PolicyEvaluation) error {
+	if status == nil {
+		return nil
+	}
+
+	mt := newTableWriter()
+
+	mt.AppendRow(table.Row{"Name", status.Material.Name})
+	mt.AppendRow(table.Row{"Type", status.Material.Type})
+	mt.AppendRow(table.Row{"Required", hBool(status.Required)})
+
+	if status.IsOutput {
+		mt.AppendRow(table.Row{"Is output", "Yes"})
+	}
+
+	if status.Value != "" {
+		v := status.Value
+		if status.Tag != "" {
+			v = fmt.Sprintf("%s:%s", v, status.Tag)
+		}
+		mt.AppendRow(table.Row{"Value", wrap.String(v, 100)})
+	}
+
+	if status.Hash != "" {
+		mt.AppendRow(table.Row{"Digest", status.Hash})
+	}
+
+	if len(status.Material.Annotations) > 0 {
+		mt.AppendRow(table.Row{"Annotations", "------"})
+		for _, a := range status.Material.Annotations {
+			value := a.Value
+			if value == "" {
+				value = NotSet
+			}
+			mt.AppendRow(table.Row{"", fmt.Sprintf("%s: %s", a.Name, value)})
+		}
+	}
+
+	if len(policyEvaluations) > 0 {
+		mt.AppendRow(table.Row{"Policies", "------"})
+		for _, evaluations := range policyEvaluations {
+			ev := evaluations
```

Comment:
This is where the bug is. The problem comes if you have a policy evaluation that doesn't have a violation. Note that we only show violations, not `ok` or `skipped` status as we do in `status`

 
<img width="586" alt="Screenshot 2025-01-13 at 22 39 38" src="https://github.com/user-attachments/assets/0d2f5567-017b-41b5-a6f3-e8cf8f095f23" />

Can we reuse as much as possible about rendering as well? For example, this is the function that handles rendering policies https://github.com/chainloop-dev/chainloop/blob/b42f2b8d479952daf4e5d6fcda14d12503766d0b/app/cli/cmd/workflow_workflow_run_describe.go#L232

If we do not do that we could run into inconsistences, like the one I can also see now where we do not show coloring output.